### ISSUE TITLE
[Bug] Engine crashed in defold 1.8.0

### ISSUE THREAD

**Issue by d954mas** [Digital-Will-Inc/wortal-sdk-defold#63]
### SDK Version

3.0.0

### Engine Version

1.8.0

### Bug Description

Engine crashed with added Wortal sdk.

I think problem with emscripten. In defold 1.8.0 emscripten was updated and need fixes is sdk.
Also progress bar logic was changed.

https://forum.defold.com/t/defold-1-8-0-has-been-released/76792

```
INFO:DLIB: Log server started on port 0 [dmloader.js:704:37](https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js)
INFO:GRAPHICS: Installed graphics device 'ADAPTER_FAMILY_OPENGL' [dmloader.js:704:37](https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js)
INFO:ENGINE: Defold Engine 1.8.0 (9141d9d) [dmloader.js:704:37](https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js)
TypeError: _stackSave is not a function
    invoke_ii https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:10897
    _main https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:10836
    callMain https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:11009
    _callMain https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:962
    _preloadAndCallMain https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:951
    runApp https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:805
    onRuntimeInitialized https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:995
    doRun https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:11032
    run https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:11042
    setTimeout handler*run https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:11038
    runCaller https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:10993
    removeRunDependency https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:274
    receiveInstance https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:366
    wasmInstantiate https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:216
    promise callback*loadAndInstantiateWasmAsync/< https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:215
    onload https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:180
    onload https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:133
    send https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:132
    load https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:190
    loadAndInstantiateWasmAsync https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:209
    instantiateWasm https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:268
    createWasm https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:375
    <anonymous> https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:10828
    loadAndRunScriptAsync https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:289
    onload https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:180
    onload https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:133
    send https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:132
    load https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:190
    loadAndRunScriptAsync https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:281
    loadWasmAsync https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:272
    onReceiveDescription https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:420
    loadArchiveDescription https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:409
    onload https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:180
    onload https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:133
    send https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:132
    load https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:190
    loadArchiveDescription https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:404
    load https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:312
    <anonymous> https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html:353
[lockdown-install.js:1:97687](moz-extension://e244ef11-2a5a-461e-8dd3-f98b0f9ac576/lockdown-install.js)
Uncaught TypeError: _stackSave is not a function
    ccall https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:10195
    onerror https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:1012
    OnErrorEventHandlerNonNull* https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:1009
    ```

### Steps to Reproduce

Build game with added sdk in defold 1.8.0+

### Relevant log output

```Shell
GET
https://dash.wortal.ai/developers/games/664c564ff80c0d0008fd0338/revisions/664dc14ff80c0d000bd05b0d
[HTTP/2 200  444ms]

GET
https://dash.wortal.ai/assets/index-11YspNVF.js
[HTTP/2 200  101ms]

GET
https://dash.wortal.ai/assets/index-8c-XmvVV.css
[HTTP/2 200  116ms]

Error in parsing value for ‘-webkit-text-size-adjust’.  Declaration dropped. index-8c-XmvVV.css:1:179
Unknown property ‘-moz-column-gap’.  Declaration dropped. index-8c-XmvVV.css:1:14428
Unknown property ‘-moz-column-gap’.  Declaration dropped. index-8c-XmvVV.css:1:14478
Unknown property ‘-moz-column-gap’.  Declaration dropped. index-8c-XmvVV.css:1:14524
Error in parsing value for ‘transition-property’.  Declaration dropped. index-8c-XmvVV.css:1:47105
Error in parsing value for ‘transition-property’.  Declaration dropped. index-8c-XmvVV.css:1:48132
Error in parsing value for ‘transition-property’.  Declaration dropped. index-8c-XmvVV.css:1:49565
GET
https://cdn.wortal.ai/fixed-static/wortal.ai.png

GET
https://cdn.wortal.ai/fixed-static/wortal.ai.png
[HTTP/2 200  98ms]

XHRGET
https://dash.wortal.ai/api/v1/developers/?page=1
[HTTP/2 200  2438ms]

XHRGET
https://dash.wortal.ai/api/v1/publishers?page=1
[HTTP/2 301  416ms]

XHRGET
https://dash.wortal.ai/api/v1/games/664c564ff80c0d0008fd0338/
[HTTP/2 200  3474ms]

XHRGET
https://dash.wortal.ai/api/v1/games/?page=1&developer=664c549ef80c0d000bd05ae5
[HTTP/2 200  470ms]

XHRGET
https://dash.wortal.ai/api/v1/publishers/?page=1
[HTTP/2 200  496ms]

GET
https://dash.wortal.ai/favicon.ico
[HTTP/2 200  492ms]

GET
https://cdn.wortal.ai/game-icons/ic_launcher.png

GET
https://cdn.wortal.ai/game-icons/ic_launcher.png
[HTTP/2 200  103ms]

XHRGET
https://dash.wortal.ai/api/v1/revisions/664dc14ff80c0d000bd05b0d
[HTTP/2 301  521ms]

XHRGET
https://dash.wortal.ai/api/v1/revisions/664dc14ff80c0d000bd05b0d/
[HTTP/2 200  1548ms]

GET
https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html
[HTTP/2 200  168ms]

GET
https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/wortal-data.js
[HTTP/2 200  111ms]

GET
https://storage.googleapis.com/cdn-wortal-ai/v2/wortal-core.js
[HTTP/2 200  381ms]

GET
https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/GameAnalytics.js
[HTTP/2 200  124ms]

GET
https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/load_bar_bg.png
[HTTP/2 200  99ms]

GET
https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/load_bar_fg.png
[HTTP/2 200  119ms]

GET
https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/modernizr-custom.js
[HTTP/2 200  108ms]

GET
https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js
[HTTP/2 200  119ms]

Unknown pseudo-class or pseudo-element ‘-webkit-full-screen’.  Ruleset ignored due to bad selector. index.html:21:24
Unknown property ‘ie-dummy’.  Declaration dropped. index.html:41:12
Unknown property ‘-moz-outline-style’.  Declaration dropped. index.html:42:22
Expected ‘none’, URL, or filter function but found ‘progid’.  Error in parsing value for ‘filter’.  Declaration dropped. index.html:115:10
 Wortal  INTERNAL  _loadCoreAsync undefined wortal-core.js:1:53564
 Wortal  Core: Initializing SDK 2.0.6 undefined wortal-core.js:1:52967
 Wortal  Core: Platform: wortal undefined wortal-core.js:1:52870
 Wortal  STATUS  Core: Loading chunks... undefined wortal-core.js:1:53263
 Wortal  INTERNAL  _loadChunksAsync undefined wortal-core.js:1:53564
GET
https://storage.googleapis.com/cdn-wortal-ai/v2/wortal-common.js
[HTTP/2 200  552ms]

GET
https://storage.googleapis.com/cdn-wortal-ai/v2/wortal-shared.js
[HTTP/2 200  450ms]

GET
https://storage.googleapis.com/cdn-wortal-ai/v2/wortal.js
[HTTP/2 200  553ms]

GET
https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/load_bar_bg.png
[HTTP/1.1 200 OK 0ms]

GET
https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/load_bar_fg.png
[HTTP/1.1 200 OK 0ms]

addProgress index.html:341:11
 Wortal  Core: Core module not loaded. Waiting for it to load before calling initializeAsync... undefined wortal-core.js:1:52870
XHRGET
https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/archive/archive_files.json
[HTTP/2 200  208ms]

XHRGET
https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/boxrobdev_wasm.js
[HTTP/2 200  105ms]

XHRGET
https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/archive/game0.projectc
[HTTP/2 200  178ms]

percentage:0.6724582699362252 index.html:322:11
percentage:0.6724582699362252 index.html:324:11
undefined lockdown-install.js:1:97687
Exception thrown, see JavaScript console dmloader.js:707:41
percentage:1.7368463893419128 index.html:322:11
percentage:1.7368463893419128 index.html:324:11
undefined lockdown-install.js:1:97687
[post-exception status] Exception thrown, see JavaScript console dmloader.js:705:40
XHRGET
https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/boxrobdev.wasm
[HTTP/2 200  168ms]

percentage:1.8100749114421264 index.html:322:11
percentage:1.8100749114421264 index.html:324:11
undefined lockdown-install.js:1:97687
Uncaught TypeError: wasmExports["rk"] is not a function
    stackSave https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:10872
    ccall https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:10195
    onerror https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:1012
    OnErrorEventHandlerNonNull* https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:1009
index.html line 289 > injectedScript:10872:42
    stackSave https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:10872
    ccall https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:10195
    onerror https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:1012
    (Async: OnErrorEventHandlerNonNull)
    <anonymous> https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:1009
 Wortal  PERFORMANCE  Chunks loaded in 490ms. undefined wortal-core.js:1:53729
 Wortal  STATUS  Core: Loading APIs... undefined wortal-core.js:1:53263
 Wortal  INTERNAL  _loadAPIsAsync undefined wortal-core.js:1:53564
 Wortal  STATUS  Session: Initializing session... undefined wortal-core.js:1:53263
XHRGET
https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/archive/game0.arci
[HTTP/2 200  164ms]

 Wortal  Game ID: 664c564ff80c0d0008fd0338 undefined wortal-core.js:1:52870
 Wortal  Session: Session initialized:  
Object { sessionID: "3d1cd97c-8745-485a-973c-203ea8920001", country: "Russia", gameID: "664c564ff80c0d0008fd0338", browser: "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0", timerID: "ab3b55dc-f002-4885-b80f-c64c754123e3", timePlayed: 0, orientation: "landscape", legacyGameID: "" }
wortal-core.js:1:52870
 Wortal  PERFORMANCE  APIs loaded in 4ms. undefined wortal-core.js:1:53729
 Wortal  INTERNAL  session.gameLoadingStart undefined wortal-core.js:1:53564
 Wortal  API  session.getPlatform undefined wortal-core.js:1:53413
 Wortal  STATUS  Core: Initializing platform... undefined wortal-core.js:1:53263
 Wortal  INTERNAL  _initializePlatformAsync undefined wortal-core.js:1:53564
 Wortal  PERFORMANCE  Platform initialized in 0ms. undefined wortal-core.js:1:53729
 Wortal  Core: Platform initialization finished, awaiting manual initialization.. undefined wortal-core.js:1:52870
GET
https://games.wortal.ai/games/wortal-sdk-2/wortal-sdk-2.css
[HTTP/2 200  96ms]

 Wortal  API  initializeAsync undefined wortal-core.js:1:53413
 Wortal  Ads: Initializing ad config.. undefined wortal-core.js:1:52870
 Wortal  STATUS  Ads: Fetching ad units from Wortal API.. undefined wortal-core.js:1:53263
 Wortal  Player initialized:  
Object { id: "8140df5d-8c28-49a4-bce3-8960be0c1aa2", name: "Player", photo: "https://storage.googleapis.com/html5gameportal.com/images/avatar.jpg", isFirstPlay: false, daysSinceFirstPlay: 1 }
wortal-core.js:1:52870
percentage:2.2558337087420006 index.html:322:11
percentage:2.2558337087420006 index.html:324:11
message received index.html:389:12
Object { target: "metamask-contentscript", data: {…} }
index.html:390:12
message received index.html:389:12
Object { target: "metamask-contentscript", data: {…} }
index.html:390:12
message received index.html:389:12
Object { target: "metamask-inpage", data: {…} }
index.html:390:12
message received index.html:389:12
Object { target: "metamask-inpage", data: {…} }
index.html:390:12
XHRGET
https://dash.wortal.ai/api/v1/ad-settings/664c564ff80c0d0008fd0338
[HTTP/2 301  446ms]

percentage:11.791462035833352 index.html:322:11
percentage:11.791462035833352 index.html:324:11
percentage:12.464490881483394 index.html:322:11
percentage:12.464490881483394 index.html:324:11
percentage:12.500200571279887 index.html:322:11
percentage:12.500200571279887 index.html:324:11
percentage:18.717958955482946 index.html:322:11
percentage:18.717958955482946 index.html:324:11
XHRGET
https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/archive/game0.arcd
[HTTP/2 200  229ms]

XHRGET
https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/archive/game1.arcd
[HTTP/2 200  316ms]

[post-exception status] Running... dmloader.js:705:40
message received index.html:389:12
Object { target: "metamask-inpage", data: {…} }
index.html:390:12
percentage:20.54201993015402 index.html:322:11
percentage:20.54201993015402 index.html:324:11
percentage:22.36608090482509 index.html:322:11
percentage:22.36608090482509 index.html:324:11
percentage:29.258858191436136 index.html:322:11
percentage:29.258858191436136 index.html:324:11
percentage:56.61977281150222 index.html:322:11
percentage:56.61977281150222 index.html:324:11
percentage:57.02323942357547 index.html:322:11
percentage:57.02323942357547 index.html:324:11
percentage:67.24645323525962 index.html:322:11
percentage:67.24645323525962 index.html:324:11
XHRGET
https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/archive/game0.dmanifest
[HTTP/2 200  155ms]

XHRGET
https://dash.wortal.ai/api/v1/ad-settings/664c564ff80c0d0008fd0338/
[HTTP/2 200  925ms]

percentage:67.70221798227254 index.html:322:11
percentage:67.70221798227254 index.html:324:11
percentage:67.98035276803743 index.html:322:11
percentage:67.98035276803743 index.html:324:11
XHRGET
https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/archive/game0.public.der
[HTTP/2 200  170ms]

percentage:67.98260723793106 index.html:322:11
percentage:67.98260723793106 index.html:324:11
[post-exception status] Running... 5 dmloader.js:705:40
GET
https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/loading.webp
[HTTP/2 200  93ms]

percentage:100 index.html:322:11
percentage:100 index.html:324:11
removeProgress index.html:346:11
INFO:DLIB: Log server started on port 0 dmloader.js:704:37
INFO:GRAPHICS: Installed graphics device 'ADAPTER_FAMILY_OPENGL' dmloader.js:704:37
INFO:ENGINE: Defold Engine 1.8.0 (9141d9d) dmloader.js:704:37
TypeError: _stackSave is not a function
    invoke_ii https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:10897
    _main https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:10836
    callMain https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:11009
    _callMain https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:962
    _preloadAndCallMain https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:951
    runApp https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:805
    onRuntimeInitialized https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:995
    doRun https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:11032
    run https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:11042
    setTimeout handler*run https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:11038
    runCaller https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:10993
    removeRunDependency https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:274
    receiveInstance https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:366
    wasmInstantiate https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:216
    promise callback*loadAndInstantiateWasmAsync/< https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:215
    onload https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:180
    onload https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:133
    send https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:132
    load https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:190
    loadAndInstantiateWasmAsync https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:209
    instantiateWasm https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:268
    createWasm https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:375
    <anonymous> https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:10828
    loadAndRunScriptAsync https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:289
    onload https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:180
    onload https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:133
    send https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:132
    load https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:190
    loadAndRunScriptAsync https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:281
    loadWasmAsync https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:272
    onReceiveDescription https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:420
    loadArchiveDescription https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:409
    onload https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:180
    onload https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:133
    send https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:132
    load https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:190
    loadArchiveDescription https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:404
    load https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:312
    <anonymous> https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html:353
lockdown-install.js:1:97687
Uncaught TypeError: _stackSave is not a function
    ccall https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/index.html line 289 > injectedScript:10195
    onerror https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:1012
    OnErrorEventHandlerNonNull* https://cdn.wortal.ai/games/664c564ff80c0d0008fd0338/664dc14ff80c0d000bd05b0d/dmloader.js:1009
index.html line 289 > injectedScript:10195:34
 Wortal  Ads: Fetching ad units from Wortal API response:  
Response { type: "cors", url: "https://dash.wortal.ai/api/v1/ad-settings/664c564ff80c0d0008fd0338/", redirected: true, status: 200, ok: true, statusText: "", headers: Headers(1), body: ReadableStream, bodyUsed: false }
wortal-core.js:1:52870
 Wortal  Ads: Fetching ad units from Wortal API response JSON:  
Object { game_id: "664c564ff80c0d0008fd0338", orientation: "any", ads: (2) […] }
wortal-core.js:1:52870
 Wortal  INTERNAL  _parseSandboxConfig undefined wortal-core.js:1:53564
 Wortal  INTERNAL  _parsePrebidConfig undefined wortal-core.js:1:53564
 Wortal  Ads: Bidders parsed. 
Object { interstitial: {…}, rewarded: {…}, timeout: 3000, failSafeTimeout: 5000, isDebugMode: true, isEnabled: false }
wortal-core.js:1:52870
 Wortal  Ads: Sandbox mode enabled. undefined wortal-core.js:1:52870
 Wortal  INTERNAL  _initializeAdSever undefined wortal-core.js:1:53564
 Wortal  INTERNAL  _initializeSandbox undefined wortal-core.js:1:53564
 Wortal  INTERNAL  _initializePrebid undefined wortal-core.js:1:53564
 Wortal  INTERNAL  _initializeAFG undefined wortal-core.js:1:53564
GET
https://games.wortal.ai/embeds/prebid-v8.37.0-pre-wortal-v1.2.js
[HTTP/2 200  102ms]

GET
https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js

Ads by Google is being shimmed by Firefox. See https://bugzilla.mozilla.org/show_bug.cgi?id=1713726 for details. sandbox eval code:1:9
<script> source URI is not allowed in this document: “https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js”. index.html:1:1
 Wortal  Ad blocker detected. AFG SDK failed to load. undefined wortal-core.js:1:53063
 Wortal  Ads: Prebid SDK loaded. undefined wortal-core.js:1:52870
 Wortal  Ads: Sandbox initialized. undefined wortal-core.js:1:52870
 Wortal  PERFORMANCE  Ads fetched in 179ms. undefined wortal-core.js:1:53729
 Wortal  Ads: Ad server initialized. undefined wortal-core.js:1:52870
 Wortal  Ads: Ad config initialized. undefined wortal-core.js:1:52870
 Wortal  INTERNAL  IAP._tryEnableIAP undefined wortal-core.js:1:53564
 Wortal  Checking for IAP compatibility.. undefined wortal-core.js:1:52870
 Wortal  IAP initialized for wortal platform. undefined wortal-core.js:1:52870
 Wortal  PERFORMANCE  Wortal SDK initialized in 2216ms. undefined wortal-core.js:1:53729
 Wortal  Core: SDK initialized for wortal platform. undefined wortal-core.js:1:52870
 Wortal  Core: SDK initialization complete. undefined wortal-core.js:1:52967
 Wortal  API  startGameAsync undefined wortal-core.js:1:53413
 Wortal  INTERNAL  session.gameLoadingStop undefined wortal-core.js:1:53564
 Wortal  INTERNAL  _sendQueuedSessionData undefined wortal-core.js:1:53564
 Wortal  Sending analytics event to Wombat 
Object { name: "GameEnd", features: {…} }
wortal-core.js:1:52870
 Wortal  Session: Sent queued session data. 
Object { name: "GameEnd", features: {…} }
wortal-core.js:1:52870
 Wortal  Sending analytics event to Wombat 
Object { name: "GameStart", features: {…} }
wortal-core.js:1:52870
POST
https://wombat.digitalwill.co.jp/wortal/events
[HTTP/1.1 204 No Content 1022ms]

POST
https://wombat.digitalwill.co.jp/wortal/events
[HTTP/1.1 204 No Content 1019ms]

 Wortal  Sending analytics event to Wombat 
Object { name: "GameEnd", features: {…} }
wortal-core.js:1:52870
POST
https://wombat.digitalwill.co.jp/wortal/events
[HTTP/1.1 204 No Content 370ms]

message received index.html:389:12
Object { target: "metamask-inpage", data: {…} }
index.html:390:12
message received index.html:389:12
Object { target: "metamask-inpage", data: {…} }
index.html:390:12
message received index.html:389:12
Object { target: "metamask-inpage", data: {…} }
index.html:390:12
message received index.html:389:12
Object { target: "metamask-inpage", data: {…} }
index.html:390:12
 Wortal  Sending analytics event to Wombat 
Object { name: "GameEnd", features: {…} }
wortal-core.js:1:52870
POST
https://wombat.digitalwill.co.jp/wortal/events
[HTTP/1.1 204 No Content 1382ms]

message received index.html:389:12
Object { target: "metamask-inpage", data: {…} }
index.html:390:12
message received index.html:389:12
Object { target: "metamask-inpage", data: {…} }
index.html:390:12
 Wortal  Sending analytics event to Wombat 
Object { name: "GameEnd", features: {…} }
wortal-core.js:1:52870
POST
https://wombat.digitalwill.co.jp/wortal/events
[HTTP/1.1 204 No Content 1071ms]

 Wortal  Sending analytics event to Wombat 
Object { name: "GameEnd", features: {…} }
wortal-core.js:1:52870
POST
https://wombat.digitalwill.co.jp/wortal/events
[HTTP/1.1 204 No Content 367ms]

 Wortal  Sending analytics event to Wombat 
Object { name: "GameEnd", features: {…} }
wortal-core.js:1:52870
POST
https://wombat.digitalwill.co.jp/wortal/events
[HTTP/1.1 204 No Content 1879ms]
```

**Comment by trdillon** [comment]
Thank you for bringing this to our attention @d954mas! We have released [v3.0.1](https://github.com/Digital-Will-Inc/wortal-sdk-defold/releases/tag/v3.0.1) which should resolve this issue. Please let us know if you have any more issues or questions.

**Comment by d954mas** [comment]
Thanks. This fix helped
